targets/XPlatCppSdk/source/code/include/playfab/OneDSAndroidHttpPlugin.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifndef DISABLE_ONEDS_API
+
+#include <playfab/PlayFabCallRequestContainer.h>
+#include <playfab/PlayFabPluginManager.h>
+#include <playfab/PlayFabAndroidHttpPlugin.h>
+
+namespace PlayFab
+{
+    /// <summary>
+    /// OneDSAndroidHttpPlugin is an https implementation to interact with OneDS (One Data Collector) services using WinHTTP.
```

Comment:
>WinHTTP. [](start = 117, length = 8)

this is not accurate here right ?

Reply:
Yes, you are right.
It should be changed to "okhttp3 though JNI"

---
In reply to: [273684820](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r273684820) [](ancestors = 273684820)